packages/browser-downloader/Dockerfile

```
@@ -0,0 +1,48 @@
+# Based on the Playwright glibc image (real Chromium + real fonts), with system
+# ffmpeg and streamlink for HLS/DASH. Runs as non-root UID 1000.
+FROM mcr.microsoft.com/playwright:v1.49.1-focal
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 \
+    PNPM_HOME=/tmp/pnpm \
+    PATH=/tmp/pnpm:$PATH \
+    NODE_ENV=production
+
+# System ffmpeg + streamlink + curl (for HEALTHCHECK).
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)